content/social-media.md

```diff
@@ -0,0 +1,198 @@
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after #.
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "[Draft] Guidelines to Make Your Social Media Platform Accessible"
+nav_title: "@@ Social Media"
+
+lang: en   # Change "en" to the translated-language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2022-10-25   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+# translators:    # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+# - name: "Jan Doe"   # Replace Jan Doe with translator name
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple translators
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with contributor name, or delete this line if none
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  repository: w3c/wai-intro-atag
+  path: content/social-media.md    # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+permalink: /standards-guidelines/atag/social-media/   # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+layout: default
+
+# NEW: 3 navigation lines below are only needed for multi-page resources where you have previous and next at the bottom. If so, un-comment them; otherwise delete these lines.
+# navigation:
+  # previous: /path/to/previous/file/
+  # next: /path/to/next/file/
+# @@SLH To Do: figure out if need to add lang here, too, and if this replaces "order" from older resources?
+
+ref: /standards-guidelines/atag/social-media/   # Translators, do not change this
+# changelog: /@@/changelog/ 
+# acknowledgements: /@@/acknowledgements/  # NEW: delete if don"t have a separate acknowledgements page. And delete it in the footer below.
+license: creative-commons   # NEW: delete if not creative-commons
+
+description:  # NEW: add a 150ish-character-description for social media   # translate the description
+# image: /content-images/wai-@@repo/social.png  # NEW: image for social media
+
+# NEW: Footer below has several options, and not all will be relevant for specific pages. (Ask Shawn if questions.)
+
+# In the footer below:
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+   <p><strong>Date:</strong> Updated 8 December 2022.<!-- First published December 2022. --></p>
```

Comment:
```suggestion
   <p><strong>Date:</strong> First published 8 December 2022.</p>
```

Reply:
Hey Daniel. I would mildly like to leave it as is, so it's consistent with the other pages now and when/if we update it later. But only mildly, so if you want to change it, that's OK with me.